src/Codec/Xlsx/Types/DataValidation.hs

```diff
@@ -0,0 +1,200 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards   #-}
+{-# LANGUAGE TemplateHaskell   #-}
+module Codec.Xlsx.Types.DataValidation where
+
+import           Control.Applicative        ((<$>))
+import           Control.Lens.TH            (makeLenses)
+import           Control.Monad              ((>=>))
+import           Data.Char                  (isSpace)
+import           Data.Default
+import qualified Data.Map                   as M
+import           Data.Maybe                 (catMaybes)
+import           Data.Monoid                ((<>))
+import           Data.Text                  (Text)
+import qualified Data.Text                  as T
+import           Text.XML                   (Node (..), Element (..))
+import           Text.XML.Cursor            (Cursor, ($/), element)
+
+import           Codec.Xlsx.Parser.Internal
+import           Codec.Xlsx.Types.Common
+import           Codec.Xlsx.Types.ConditionalFormatting
+import           Codec.Xlsx.Writer.Internal
+
+-- See 18.18.21 "ST_DataValidationType (Data Validation Type)" (p. 2440/2450)
+data ValidationType
+    = ValidationTypeNone
+    | ValidationTypeCustom     Formula
+    | ValidationTypeDate       OperatorExpression
```

Comment:
this isn't quite correct `ST_DataValidationOperator` != `ST_ConditionalFormattingOperator` unfortunately :(